code/datums/ai/objects/mod.dm

```diff
@@ -0,0 +1,45 @@
+/datum/ai_controller/mod
+	blackboard = list(
+		BB_MOD_TARGET = null,
+		BB_MOD_IMPLANT = null,
+	)
```

Comment:
no need to include these as nulls by default, the neat thing is that checking a blackboard for `BB_MOD_TARGET` for example will return null if it isn't in the blackboard